hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -432,3 +432,36 @@ tests:
       - eval_time: 1m
         alertname: VMCannotBeEvicted
         exp_alerts: []
+
+  # Test recording rule
+  - interval: 1m
+    input_series:
+      # in reality there are many labels on these metrics
+      # they are the same expect the ones containing vm name like "name" in the example below
```

Comment:
@erkanerol 
in -> In
expect - > except

Reply:
fixed